### PR TITLE
added mariadb build steps to dev image

### PR DESCRIPTION
**Pull request by ZStriker19**
## Description
This adds mariadb to the test docker container so that the tests for adding mariadb integration pass: https://github.com/DataDog/dd-trace-py/pull/2570

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

File: Dockerfile.buster [hunk 1]
```diff
@@ -54,8 +54,13 @@ RUN \
   && echo "deb-src https://deb.nodesource.com/${NODE_VERSION} buster main" | tee -a /etc/apt/sources.list.d/nodesource.list \
   && apt-get update \
   && apt-get install -y --no-install-recommends nodejs \
+  && apt-get install -y mariadb-server \
```
Comment (brettlangdon) [inline Dockerfile.buster:57]: do we need mariadb-server here or just `libmariadb3`?

we'll only want to install what is needed for building the Python package.

Comment (ZStriker19) [reply Dockerfile.buster]: Yeah, you're right, just tested and that was unnecessary to run the tests. Thanks for the catch.

File: Dockerfile.buster [hunk 2]
```diff
@@ -54,8 +54,12 @@ RUN \
   && echo "deb-src https://deb.nodesource.com/${NODE_VERSION} buster main" | tee -a /etc/apt/sources.list.d/nodesource.list \
   && apt-get update \
   && apt-get install -y --no-install-recommends nodejs \
+  && curl -sS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | bash \
```
Comment (brettlangdon) [inline Dockerfile.buster:57]: Can you add a comment with a link to the MariaDB instructions to install like this?

I'd prefer to not pipe a script into bash, but from looking at their docs they don't offer any alternative instructions, we'd have to pull the steps out of that script if we wanted to do that

Comment (Kyle-Verhoog) [reply Dockerfile.buster:57]: IIRC it is from their docs but ➕  to adding a link to them for future reference.

**Review by Kyle-Verhoog — COMMENTED** [review]
lgtm with @brettlangdon's comment addressed

File: Dockerfile.buster (hunk 2 above)
Comment (ZStriker19) [reply Dockerfile.buster:57]: https://mariadb.com/kb/en/mariadb-package-repository-setup-and-usage/ but per Kyle's suggestion let's try removing.

File: Dockerfile.buster [hunk 3]
```diff
@@ -54,8 +54,11 @@ RUN \
   && echo "deb-src https://deb.nodesource.com/${NODE_VERSION} buster main" | tee -a /etc/apt/sources.list.d/nodesource.list \
   && apt-get update \
   && apt-get install -y --no-install-recommends nodejs \
+  && apt-get install -y libmariadb3 \
   # Cleaning up apt cache space
-  && rm -rf /var/lib/apt/lists/*
+  && rm -rf /var/lib/apt/lists/* \
```
Comment (Kyle-Verhoog) [inline Dockerfile.buster:59]:
```suggestion
  && rm -rf /var/lib/apt/lists/*
```

File: Dockerfile.buster [hunk 4]
```diff
@@ -54,10 +54,13 @@ RUN \
   && echo "deb-src https://deb.nodesource.com/${NODE_VERSION} buster main" | tee -a /etc/apt/sources.list.d/nodesource.list \
   && apt-get update \
   && apt-get install -y --no-install-recommends nodejs \
+  && apt-get install -y libmariadb3 \
   # Cleaning up apt cache space
   && rm -rf /var/lib/apt/lists/*
 
 
+
+
```
Comment (brettlangdon) [inline Dockerfile.buster:63]: These new blank lines are unnecessary

```suggestion
```

**Review by brettlangdon — DISMISSED** [review]
other than extra lines, lgtm

**Review by Kyle-Verhoog — APPROVED** [review]

**Review by brettlangdon — APPROVED** [review]

**Pull request merged**